index.js

```diff
@@ -28,36 +26,36 @@ module.exports = function(content) {
     this.emitWarning("DEPRECATED. Configure optipng's optimizationLevel option in its own options. (optipng.optimizationLevel)");
   }
 
-  var callback = this.async(),
-    called = false;
+  const callback = this.async();
+  let called = false;
 
   if (this.debug === true && options.bypassOnDebug === true) {
     // Bypass processing while on watch mode
     return callback(null, content);
-  } else {
```

Comment:
No need for this `else`, since the `if` contains a return. 
we save one level of indentation / code complexity